.editorconfig

```diff
@@ -1,50 +1,50 @@
-# Suppress: EC112
-# top-most EditorConfig file
-root = true
```

Comment:
Something is stil not right. It has 58 changed files. I think you still have some problems with the spacing. Can you check?